.github/workflows/build.yml

```diff
@@ -34,12 +34,12 @@ jobs:
       with:
         name: NuGet packages
         path: |
-          ${{ github.workspace }}/artifacts/*.nupkg
-          ${{ github.workspace }}/artifacts/*.snupkg
+          artifacts/*.nupkg
+          artifacts/*.snupkg
 
     - name: Push NuGet packages to NuGet
       if: ${{ startsWith(github.ref, 'refs/tags/v') }} # Only for a tag
-      run: dotnet nuget push "artifacts/*.nupkg" --skip-duplicate --api-key ${{ secrets.NUGET_KEY }} --source https://api.nuget.org/v3/index.json
+      run: dotnet nuget push "artifacts/**.nupkg" --skip-duplicate --api-key ${{ secrets.NUGET_KEY }} --source https://api.nuget.org/v3/index.json
```

Comment:
Was poking around trying to find info on this and this was the best I could find: https://github.com/NuGet/Home/issues/4393#issuecomment-667618120

Based on that comment, it *seems* like `"artifacts/*.nupkg"` is supposed to work so it's super weird that it's not.

Reply:
🤷‍♂️ Yeah, it's strange.  But I was able to reproduce it locally.